### DISCUSSION TITLE
Tooltip floating element is not positioned relative to parent.

### DISCUSSION THREAD

**Discussion by kishoreio** [floating-ui/floating-ui#2441] [Q&A]
I tried building a tooltip using floating-ui in react. When I use div element as a reference element, the floating element is not placed exactly where it supposed to be.

Here's the code - [https://codesandbox.io/s/optimistic-hoover-ny5dqt?file=/src/App.js:0-982](https://codesandbox.io/s/optimistic-hoover-ny5dqt?file=/src/App.js:0-982)

When I hover on the div element, the floating element is showing in right that overflows. I need the element positioned correctly next to div element.

**Comment by atomiks** [comment]
That's just because the div is a block and spans the entire width of the page. You're only looking at the text inside it, not the element block. Needs `display: inline-block`.
